test/README.md

```diff
@@ -0,0 +1,144 @@
+
+**********This file describes the work done and steps to perform test on the robot localization package.**********
+
+"https://github.com/Rohita83/robot_localization"
```

Comment:
Mind of we remove this line for the main repo?

Reply:
Done in https://github.com/cra-ros-pkg/robot_localization/commit/87f41f0655e31fb247f06dc24f31a4d3fc7af83b